test/secrets/backfiller_dburl

```diff
@@ -0,0 +1 @@
+mysql+tcp://sa@boulder-mysql:3306/boulder_sa_integration
```

Comment:
Not clear to me if we should go through the hoops of creating a new DB user that has `GRANT`'s to read `certificates` and `certificateStatus` and to `UPDATE` `certificateStatus`. Using the SA user for now in the example dbURL file.